firmware/applications/SignBuddy/disco_board.h

```diff
@@ -28,48 +28,54 @@
 
 // To do: Configure second uart on discovery for sending to ble
 /* BLE UART */
-#define BLE_UART                  LPUART1
-#define BLE_UART_CONFIG           LL_LPUART_InitTypeDef
-#define BLE_UART_IRQn             LPUART1_IRQn
-#define BLE_UART_IRQHandler       LPUART1_IRQHandler
-#define BLE_UART_RX_PIN           LL_GPIO_PIN_0
-#define BLE_UART_TX_PIN           LL_GPIO_PIN_1
-#define BLE_UART_GPIO_PORT        GPIOC
-#define BLE_UART_GPIO_AF          LL_GPIO_AF_8
-#define BLE_UART_DATAWIDTH        LL_LPUART_DATAWIDTH_8B
-#define BLE_UART_STOPBITS         LL_LPUART_STOPBITS_1
-#define BLE_UART_PARITY           LL_LPUART_PARITY_NONE
-#define BLE_UART_DIRECTION        LL_LPUART_DIRECTION_TX_RX
-#define BLE_UART_FLOWCTRL         LL_LPUART_HWCONTROL_NONE
-#define BLE_UART_Init             LL_LPUART_Init
-#define BLE_UART_Enable           LL_LPUART_Enable
-#define BLE_UART_RX               LL_LPUART_ReceiveData8
-#define BLE_UART_TX               LL_LPUART_TransmitData8
-#define BLE_UART_CLK_SRC()        LL_RCC_SetLPUARTClockSource(LL_RCC_LPUART1_CLKSOURCE_HSI)
-#define BLE_UART_CLK_EN()         LL_APB1_GRP2_EnableClock(LL_APB1_GRP2_PERIPH_LPUART1)
-#define BLE_UART_GPIO_CLK_EN()    LL_AHB2_GRP1_EnableClock(LL_AHB2_GRP1_PERIPH_GPIOC)
+#define BLE_UART                      LPUART1
+#define BLE_UART_CONFIG               LL_LPUART_InitTypeDef
+#define BLE_UART_IRQn                 LPUART1_IRQn
+#define BLE_UART_IRQHandler           LPUART1_IRQHandler
+#define BLE_UART_RX_PIN               LL_GPIO_PIN_0
+#define BLE_UART_TX_PIN               LL_GPIO_PIN_1
+#define BLE_UART_GPIO_PORT            GPIOC
```

Comment:
I don't think we need to abstract all of these in the future. It might be nice to have but it seems like a lot of work for something that's not likely to change

Reply:
what do you think?

Reply:
We might as well leave it now, if we want to test the bluetooth module with the discovery we'll have to use usart